BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -354,19 +354,38 @@ extension AttributedLabel {
                 return []
             }
 
-            func alignmentMultiplier() -> CGFloat {
-                // If a paragraph style spans the range of the text, use its alignment.
-                // Otherwise, fall back to `.natural`.
-                let alignment: NSTextAlignment = {
-                    var range: NSRange = NSRange(location: -1, length: 0)
-                    let style = textStorage.attribute(.paragraphStyle, at: 0, effectiveRange: &range)
-
-                    guard let style = style as? NSParagraphStyle, range == textStorage.entireRange else {
-                        return .natural
+            /// The below positioning calculation assumes that there is only one
+            /// alignment within the label, so verify that is the case.
+            func alignment() -> NSTextAlignment {
```

Comment:
@watt This is what I ended up coming up with to (a) validate we only have one alignment, and (b) use that alignment (vs just `textAlignment`) for link calculation, since we don't actually know `textAlignment`'s heuristic (though its probably basically just what we do here, in practice, I assume).

Reply:
@watt Ping for a review here when you get a chance!

Reply:
@watt Ping on this if you have time this week

Reply:
Sounds fine to me. Can you add a bit to this function's doc comment about the `textAlignment` fallback?

Reply:
👍

Reply:
- [ ] Update docs